github-upload-secrets

```diff
@@ -98,7 +98,7 @@ def upload_secrets(
     for name, secret in secrets.items():
```

Comment:
The "ruff: enable preview mode" commit says "run ruff as mypy", it certainly means "ruff and mypy"